### ISSUE TITLE
Scroll Workspace name on secondary monitor not switching workspace

### ISSUE THREAD

**Issue by buzzcz** [paperwm/PaperWM#814]
**Describe the bug**
When scrolling the mouse wheel on the Workspace name on the secondary monitor, the workspace is not switched. It works only on the primary monitor.

**To Reproduce**
1. Have two monitors with PaperWM
2. Scroll the mouse wheel on the Workspace name on the secondary monitor
3. Nothing happens

**Expected behavior**
When scrolling the mouse wheel on the Workspace name on the secondary monitor, the workspace switcher should appear as it does on the primary monitor.

**System information:**

```
Distribution: Fedora Linux
GNOME Shell 44.10
Display server: Wayland
PaperWM branch/tag: fatal: not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
fatal: not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
fatal: not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
PaperWM commit: fatal: not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
Enabled extensions:
- auto-move-windows@gnome-shell-extensions.gcampax.github.com
- always-indicator@martin.zurowietz.de
- bluetooth-battery@michalw.github.com
- blur-my-shell@aunetx
- dash-to-dock@micxgx.gmail.com
- paperwm@paperwm.github.com
- background-logo@fedorahosted.org
- GPaste@gnome-shell-extensions.gnome.org
```

**Additional context**
PaperWM version from extensions.gnome.org

**Comment by jtaala** [comment]
Hey @buzzcz,

Just a note that this is expected - the "topbar" on secondary monitors isn't actually a Gnome topbar (it's a fake topbar) as Gnome only supports one topbar (on primary).  I suppose it can be added though.

Also, seeing that' you're on Gnome 44 (which isn't actively supported/developed for, and doesn't have the latest features/fixes etc.), we'd implement this change on in Gnome 45/46 and then look at backporting it to 44.

Jay.

**Comment by buzzcz** [comment]
I see, didn't realize that. Thanks for explaining.
Regarding Gnome 44, it's ok for me if it's not backported. I hope I'll upgrade my Fedora installation soon or at least once F40 is out of Beta.

Thanks for the great extension and good luck :+1:

**Comment by jtaala** [comment]
Hey @buzzcz,

I've pushed up a PR to fix this issue (see #833) - although this is on our main release branch so would be for Gnome 45/46 only.

**Comment by buzzcz** [comment]
Great to hear that. Looking forward to upgrading and checking it out. Thank you!
